src/core/CMakeLists.txt

```diff
@@ -1,17 +0,0 @@
-add_subdirectory(core)
```

Comment:
Now the core directory has been completely removed?

Reply:
Yes.